compiler-rt/test/asan/TestCases/use-after-scope-loop-bug.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx_asan -O1 %s -o %t && not %run %t 2>&1 | FileCheck %s
+// RUN: %clangxx_asan %if MSVC %{ /Od %} %else %{ /O1 %} %s -o %t && not %run %t 2>&1 | FileCheck %s
```

Comment:
This was using `-O1` which works both with MSVC, clang-cl and the clang driver, don't switch to `/O1` which isn't accepted by the GNU style clang driver.